_common/config/settings.py

```diff
@@ -0,0 +1,19 @@
+import os
+from pydantic_settings import BaseSettings
+from typing import Optional
+from pathlib import Path # pathlib 추가
+
+PROJECT_ROOT_DIR = Path(__file__).resolve().parent.parent.parent
+DOTENV_PATH = PROJECT_ROOT_DIR / ".env"
+
+class CommonSettings(BaseSettings):
+    # 데이터베이스 설정
+    DATABASE_URL: str = os.getenv("DATABASE_URL", "postgresql://user:pass@host:port/db")
```

Comment:
좋은 기본값 설정입니다. 다만, 환경변수 키는 좀 더 명확히 구분되도록 다음처럼 바꾸는 건 어떨까요? 
저도 "CRAWLER_SERVICE_MONGODB_URI" 이렇게 사용중이에요
 ```suggestion 
 DATABASE_URL: str = os.getenv("API_SERVICE_POSTGREDB_URL", "postgresql://user:pass@host:port/db")
 ```

Reply:
좋습니다😃 다만 POSTGREDB_URL 앞에 API_SERVICE를 붙일 이유는 없는 것 같아 
DATABASE_URL > POSTGREDB_URL로 수정하였습니다! :) 
MongoDB도 크롤링 단계에서만 사용하는게 아니고, MongoDB > PostgreSQL이 연결될 때도 사용될 것 같으니 MONGODB_URI로 해도 되지 않을까 싶긴하네용! 
의견 감사합니당👍🏻